File: src/main/java/ninekothecat/catconomy/interfaces/ICatEconomyCommandAutoComplete.java
package ninekothecat.catconomy.interfaces;

import org.bukkit.command.Command;
import org.bukkit.command.CommandSender;
import org.jetbrains.annotations.NotNull;

import java.util.List;

public interface ICatEconomyCommandAutoComplete {
    List<String> onTabComplete(@NotNull CommandSender sender, @NotNull Command command, @NotNull String alias, String[] args);
}

File: src/main/kotlin/ninekothecat/catconomy/interfaces/ICatEconomyCommandAutoComplete.kt
package ninekothecat.catconomy.interfaces

import org.bukkit.command.Command
import org.bukkit.command.CommandSender

interface ICatEconomyCommandAutoComplete {
    fun onTabComplete(sender: CommandSender, command: Command, alias: String, args: Array<String>?): List<String>?
}